Include/objimpl.h

```diff
@@ -251,9 +251,9 @@ PyAPI_FUNC(PyVarObject *) _PyObject_GC_Resize(PyVarObject *, Py_ssize_t);
 #ifndef Py_LIMITED_API
 typedef union _gc_head {
     struct {
-        union _gc_head *gc_next;
-        union _gc_head *gc_prev;
-        Py_ssize_t gc_refs;
+        union _gc_head *gc_next;  // NULL means the object is not tracked
+        uintptr_t gc_prev;  // Pointer to previous object in the list.
+                            // Lowest three bits are used for flags.
     } gc;
     double dummy;  /* force worst-case alignment */
```

Comment:
Hum, I think that alignment using sizeof(double) comes the C standard. If I'm right, it might help to document it here :-)

Reply:
It is separated issue, see https://bugs.python.org/issue33589

Reply:
Uh, it's not separated issue.  Since I this pull request removes last `Py_ssize_t` member from gc, there are only two pointers (or integer having same size to pointer) in the struct.
It must be aligned 8byte on all 32bit and 64bit platforms. dummy can be removed!

Reply:
I removed dummy, and stop using union too.